packages/hdwallet-xdefi/src/xdefi.ts

```diff
@@ -37,6 +37,10 @@ export class XDEFIHDWalletInfo implements core.HDWalletInfo, core.ETHWalletInfo
     return false;
   }
 
+  public supportsBip44Accounts(): boolean {
+    return true; // probably?
```

Comment:
requires testing

Reply:
this needs to be false - it supports multiple accounts but not at the same time.

only native + keepkey support them at the same time currently

Reply:
🙏